includes/Core/Authentication/Verification_File.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Class Verification_File
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Authentication;
+
+use Google\Site_Kit\Core\Storage\User_Options;
+
+/**
+ * Class representing the site verification file for a user.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+final class Verification_File {
+
+	/**
+	 * User option key.
+	 */
+	const OPTION = 'googlesitekit_site_verification_file';
```

Comment:
This option would store the "file token", not the entire filename (i.e. it would store the filename minus `.html`).